.schema/openapi/patches/oauth2.yaml

```diff
@@ -1,4 +1,4 @@
 - op: remove
-  path: /components/schemas/consentRequestSession/properties/access_token/type
+  path: /components/schemas/consentRequestSession/properties/access_token/additionalProperties
 - op: remove
-  path: /components/schemas/consentRequestSession/properties/id_token/type
+  path: /components/schemas/consentRequestSession/properties/id_token/additionalProperties
```

Comment:
Could you please keep the original definition and add a remove patch for the additionalProperties on top? :)

Reply:
Sure, my apologies

Reply:
Without both parameters the type becomes `interface{}`, see https://github.com/ory/hydra/pull/3076/commits/e8d336b4307b604ce159d91680f9435b4c1a3dca

Reply:
Yes, that’s perfect